src/reducers/patients.js

```diff
@@ -0,0 +1,13 @@
+import { FETCH_PATIENTS_FULFILLED } from '../actions/types';
+
+export default function(state = {patientEntries: [], meta:{}}, action)  {
+  switch (action.type) {
+    case FETCH_PATIENTS_FULFILLED:
+      const patientEntries = action.payload.data.entry.map((e) => e.resource);
+      const { total, link } = action.payload.data;
+      const meta = { total,link };
```

Comment:
No it can't. `meta.total` is the total number of patients in the system matching that criteria. The length of patients is the number of patients in this page of data.